src/components/Content/Content.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import { ReactElement, useState } from "react";
+import { ContentProps, ContentType } from "./ContentTypes";
+
+// Components
+import Button from "../Button";
+import Chip from "../Chip";
+import {
+	MainWrapper,
+	Title,
+	TitleWrapper,
+	ButtonsWrapper,
+	Label,
+	Value,
+	LabelValueWrapper,
+	Paragraph,
+	ChipsWrapper,
+	LabelWrapper,
+	ValueFile,
+	ContentRow,
+	ContentColumn,
+	ParagraphWrapper,
+} from "./Content.styled";
+import EditIcon from "@mui/icons-material/Edit";
+import AddIcon from "@mui/icons-material/Add";
+
+const Content = (props: ContentProps): ReactElement => {
+	const { title, onClickEdit, content, onClickAdd } = props;
+	const [showOrHide, setShowOrHide] = useState(content.length < 3);
+
+	/**
+	 * Toggles the state use to show or hide the content.
+	 */
+	const showOrHideDetails = () => {
+		setShowOrHide(!showOrHide);
+	};
+
+	const contentForDisplay = showOrHide ? content : content.slice(0, 3);
+
+	/**
+	 * Renders the content of type "labelValue"
+	 * @param contentItem 
+	 * @returns The content with a sctructure of label: value
+	 */
+	const labelValueContent = (contentItem: ContentType) => (
+		<LabelValueWrapper>
+			<Label>{contentItem.label}:</Label>
+			<Value>{contentItem.value}</Value>
+		</LabelValueWrapper>
+	);
+
+	/**
+	 * Renders the content of type "paragraph"
+	 * @param contentItem 
+	 * @returns A paragraph with the value of the content
+	 */
+	const paragraphContent = (contentItem: ContentType) => (
+		<ParagraphWrapper>
+			<Label>{contentItem.label}:</Label>
+			<Paragraph>{contentItem.value}</Paragraph>
+		</ParagraphWrapper>
+	);
+
+	/**
+	 * Renders the content of type "tags"
+	 * @param contentItem 
+	 * @returns The a lists of tags/chips.
+	 */
+	const tagsContent = (contentItem: ContentType) => {
+		if (!Array.isArray(contentItem.value)) {
+			return;
+		}
+
+		return (
+			<>
+				<Label>{contentItem.label}:</Label>
+				<ChipsWrapper>
+					{contentItem.value.map((value, idx) => (
+						<Chip key={`${idx}-${value}`} label={value} />
+					))}
+				</ChipsWrapper>
+			</>
+		);
+	};
+
+	/**
+	 * Renders the content of type "file"
+	 * @param contentItem 
+	 * @returns The info of the file passed within the label
+	 * and value attributes and it renders an icon if any.
+	 */
+	const fileContent = (contentItem: ContentType) => {
+		const FileIcon = contentItem?.icon;
+
+		return (
+			<>
+				<LabelWrapper>
+					{FileIcon && <FileIcon />}
+					<Label>{contentItem.label}:</Label>
+				</LabelWrapper>
+				<ValueFile>{contentItem.value}</ValueFile>
```

Comment:
Same comment as with the paragraph.

Reply:
This can be done since both are `<span>` I removed the ValueFile styled component and use the Value with a prop that defines the styles for each case.